velero-2-mixin/targets.libsonnet

```diff
@@ -100,5 +100,115 @@ local utils = commonlib.utils {
       )
       + prometheusQuery.withLegendFormat('%s - failure' % utils.labelsToPanelLegend(this.config.legendLabels)),
 
+    lastBackupStatus:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'velero_backup_last_status{%(queriesSelector)s}' % vars
+      )
+      + prometheusQuery.withLegendFormat('Backups'),
+
+    restoreValidationFailure:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'sum(increase(velero_restore_validation_failed_total{%(queriesSelector)s}[$__interval:]))' % vars
+      )
+      + prometheusQuery.withLegendFormat('Failure'),
+    backupSuccess:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'increase(velero_backup_success_total{%(queriesSelector)s}[$__interval:])' % vars
+      )
+      + prometheusQuery.withLegendFormat('success'),
+    backupFailure:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'increase(velero_backup_failure_total{%(queriesSelector)s}[$__interval:])' % vars
+      )
+      + prometheusQuery.withLegendFormat('failure'),
+
+    backupSuccessRate:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'rate(velero_backup_success_total{%(queriesSelector)s}[$__rate_interval]) / clamp_min(rate(velero_backup_attempt_total{%(queriesSelector)s}[$__rate_interval]),0.001)' % vars
+      )
+      + prometheusQuery.withLegendFormat('success'),
```

Comment:
I realized that grafana dropped an empty attribute that comes alongisde metrics that can be populated: schedule with this realization I have a few choices.

Include cluster - instance - schedule
instance - schedule
schedule
Since these are all single selections for now. I think choice number 3 would be best... but for now i went with choice number 1. Let me know what you think. I will try out a couple of the other options and try to see which looks best.